.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,9 +6,13 @@ about: Report a bug in TSLint
 
 ### Bug Report
 
-- __TSLint version__:
-- __TypeScript version__:
-- __Running TSLint via__: (pick one) CLI / Node.js API / VSCode / grunt-tslint / Atom / Visual Studio / etc
+-   **TSLint version**:
+-   **TypeScript version**:
+-   **Running TSLint via**: (pick one) CLI / Node.js API / VSCode / grunt-tslint / Atom / Visual Studio / etc
+
+#### Reproduction using [TSLint Playground](https://palantir.github.io/tslint-playground/)
+
+Bug reports for rules should include a reproduction using the tslint-playground website. Click the `Save to URL` button and paste the URL here.
```

Comment:
maybe this should be in a `<!-- comment -->` to avoid polluting issue descriptions?